include/mgmt_conn.h

```diff
@@ -62,6 +62,7 @@ typedef struct uzfs_mgmt_conn {
 	int		conn_procn;	// bytes already read/written
 	zvol_io_hdr_t	*conn_hdr;	// header of currently processed cmd
 	time_t		conn_last_connect;  // time of last attempted connect()
+	boolean_t disabled;	// connection is disabled due to internal errors
```

Comment:
lets update comment saying 'conn is disabled due to reasons like read only zvol'
indentation missing ?
changing flag name to connect_disabled makes sense?

Reply:
done.